google/ggrc-core
# Fix issue mapper tests

# Issue description
The tests left old acl entries in the database when running. The test
itself was not actually broken, but was just messy and hard to debug
with too many items remaining in the database.


# Steps to test the changes

Run that single test file multiple times and check the ACL table for data. There should be exactly the same amount of data after each run, not adding new data on top of previous runs.

# Solution description

Add proper cleanup to setup stage and add a bit more info to test docstring.

# Sanity checklist

- [ ] N/A - App code was not touched. I have clicked through the app to make sure my changes work and not break the app.



# PR Review checklist
